packages/angularjs-extras/src/controllers/panel.js

```diff
@@ -34,8 +34,8 @@
 
       // Digest Model Bindings
       $scope.model = null
-      if ($element.attr('ng-model')) {
-        $scope.$parent.$watch($element.attr('ng-model'), function (model) {
+      if ($element.attr('data-ng-model')) {
+        $scope.$parent.$watch($element.attr('data-ng-model'), function (model) {
```

Comment:
This should check for both, as technically both are valid possibilities.

Reply:
@alexgurrola In an effort to simplify and standardize (we don't want to support multiple variations) I thought it was better to use only the valid version, only `data-*` is valid by the standards. So it makes no sense to allow a non-standard `ng-x` as well. If someone tries to use ng-x and it doesn't work, it should be obvious that they didn't use a valid attribute. Right?